draft-ietf-quic-transport.md

```diff
@@ -3218,6 +3218,19 @@ value of the connection ID changed.  An endpoint that is sending packets with a
 zero-length Destination Connection ID MUST treat receipt of a NEW_CONNECTION_ID
 frame as a connection error of type PROTOCOL_VIOLATION.
 
+Transmission errors, time-outs and retransmission may cause the same
+NEW_CONNECTION_ID frame to be received multiple time. Additional
+transmissions SHOULD be ignored and MUST NOT cause a connection
+error.
+
+If an endpoint receives a NEW_CONNECTION_ID frame that repeats the same
+connection ID as a previous NEW_CONNECTION_ID frame but with a different
+Stateless Reset Token or a different Sequence, the endpoint MUST
+treat that receipt as a connection error of type PROTOCOL_VIOLATION.
+Similarly, if an endpoint receives a NEW_CONNECTION_ID frame that repeats
```

Comment:
That MUST is hard to achieve if the context has been erased meanwhile. Surely the intention is not to log ID's forever?

Reply:
As @mikkelfj says, I think that we only need to say "MAY" for these and let nature take its course.  A "MAY treat as a protocol error" turns into a fairly firm requirement not to send duplicates.